### PR TITLE
Edited August first details and reformatted

### PR DESCRIPTION
**Pull request by deadlion2ttip**
Hey all! I thought I'd update August First's entry to let you all know what upstanding community members they are. I also reformatted a few things.

- Chris Griffin

### CODE REVIEW

File: august-first.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # August First
+#HOURS
+**Monday – Saturday: 7:30am – 4:00 pm**
 
 **Address** 149 S Champlain St, Burlington, VT 05401
 
 **Phone Number** (802)540-0060
 
+**Web Page URL:** (augustfirstvt.com)
```
Comment (alexch) [inline august-first.md]: the URL should use angle brackets, not parentheses, so the link gets 🔥 hot 🔥

**Comment by alexch** [comment]
There's a merge conflict on this PR; it would be a good exercise to resolve it. Ask me or another teacher for help if you get confused or stuck. Or maybe we can do it in class Friday morning.

**Comment by deadlion2ttip** [comment]
I resolved those conflicts by picking the more detailed content to display.

**Comment by sspencervt** [comment]
@alexch @Dangeranger Hey all merge conflicts are resolved, can you merge?

**Comment by Dangeranger** [comment]
All good.
👍 🎉 
Thanks @sspencervt @deadlion2ttip

**Pull request merged**